flamingo/services/alias_engine.py

```diff
@@ -33,7 +33,7 @@ def replace(self, virtual_value):
         for alias_to in aliases_to:
             try:
                 replace_with = self.replacements[alias_to]
-                new_value = new_value.replace("${%s}" % alias_to, replace_with)
+                new_value = new_value.replace(f"${{alias_to}}", replace_with)
```

Comment:
Did you test this?
String formatting on Cloud Build triggers are **very** finicky.

Reply:
Oops, indeed it was missing a pair of braces.

Now it should be ok:
```
>>> alias_to = "some_var"
>>> f"${{{alias_to}}}"
'${some_var}'
```